manual/Creating_a_table/Creating_a_sharded_table/Creating_a_sharded_table.md

```diff
@@ -0,0 +1,53 @@
+# manual Creating a Sharded Table
+
+Manticore allows for the creation of **sharded tables**, which act like a special distributed table containing multiple tables that represent shards. This feature can be useful if you need to scale data. You can create a local sharded table and a replicated sharded table when multiple clusters are required.
+
+#### Create a Local Sharded Table
+
+To create a local sharded table, you can create a table as normal but also add two parameters: `shards='x'` and `rf='1'`. `shards` represents the number of local tables that will be created and represent shards. `rf` is the replication factor, and for a single server environment, you need to use 1 here.
+
+Here's an example to create a table that will contain 10 shards, and all data should be automatically distributed:
+
+```sql
+CREATE TABLE local_sharded shards='10' rf='1'
+```
+
+After this query, you will get multiple tables that represent shards and a final distributed table with the name `local_sharded` that you can use to insert and read data, and the distribution logic will be handled by the Manticore Search daemon.
```

Comment:
> get multiple tables that represent shards

Since the tables are hidden by default, I think it's important to let the user know. Otherwise, they might get confused when they run `show tables` and don’t see anything listed.